istio/src/main/scala/io/buoyant/k8s/istio/CurrentIstioPath.scala

```diff
@@ -20,7 +20,7 @@ object CurrentIstioPath {
   }
 
   /**
-   * Returns the target service requested based on the supplied [[Path]].
```

Comment:
Maybe add backtick (`) characters around the former links so they'll be typeset as code?